smoke/helpers.go

```diff
@@ -9,13 +10,34 @@ import (
 	. "github.com/onsi/gomega/gexec"
 )
 
+var DEFAULT_TIMEOUT = 30 * time.Second
+
+func SetBackend(appName string) {
+	config := GetConfig()
+	if config.Backend == "diego" {
+		EnableDiego(appName)
+	} else if config.Backend == "dea" {
+		DisableDiego(appName)
+	}
+}
+
 func EnableDiego(appName string) {
```

Comment:
Hi @joek,
We discussed whether it would be worthwhile to add a skip logic to smoke tests similar to the logic [we have in cats](https://github.com/cloudfoundry/cf-acceptance-tests/blob/2a3809051d27918e1064dad7e7d45ed2e4aebf75/route_services/route_services_test.go#L22), but since smoke tests are a very small suite of tests in comparison, this feels heavy handed.

Given that, we are going to merge this pull request as-is.

Thanks,
Al & Rob